validator/accounts/accounts_exit.go

```diff
@@ -222,9 +220,38 @@ func performExit(cliCtx *cli.Context, cfg performExitCfg) ([]string, error) {
 			}
 		}
 		if !found {
+			rawExitedKeys = append(rawExitedKeys, key)
 			formattedExitedKeys = append(formattedExitedKeys, cfg.formattedPubKeys[i])
 		}
 	}
 
-	return formattedExitedKeys, nil
+	return rawExitedKeys, formattedExitedKeys, nil
+}
+
+func displayExitInfo(rawExitedKeys [][]byte, trimmedExitedKeys []string) {
+	if len(rawExitedKeys) > 0 {
+		urlFormattedPubKeys := make([]string, len(rawExitedKeys))
+		for i, key := range rawExitedKeys {
+			var baseUrl string
+			if params.BeaconConfig().ConfigName == "pyrmont" {
```

Comment:
Let's define this in the `param` pkg via a map to avoid fragmentation down the line

Reply:
agreed